miniwallet/contracts/mocks/TestERC1155.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+/// @custom:security-contact dev@modulo.so
+contract TestERC1155 is ERC1155, AccessControl {
+    bytes32 public constant URI_SETTER_ROLE = keccak256("URI_SETTER_ROLE");
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+
+    constructor(uint256[] memory tokenIds, uint256[] memory amounts)
+        ERC1155("ipfs://testerc1155.modulo.so/")
+    {
+        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        _grantRole(URI_SETTER_ROLE, msg.sender);
+        _grantRole(MINTER_ROLE, msg.sender);
+        for (uint32 i = 0; i < tokenIds.length; i++) {
+            mint(msg.sender, tokenIds[i], amounts[i], "");
+        }
+    }
+
+    function setURI(string memory newuri) public onlyRole(URI_SETTER_ROLE) {
+        _setURI(newuri);
+    }
+
+    function mint(
+        address account,
+        uint256 id,
+        uint256 amount,
+        bytes memory data
+    ) public onlyRole(MINTER_ROLE) {
+        _mint(account, id, amount, data);
+    }
+
+    function mintBatch(
+        address to,
+        uint256[] memory ids,
+        uint256[] memory amounts,
+        bytes memory data
+    ) public onlyRole(MINTER_ROLE) {
+        _mintBatch(to, ids, amounts, data);
+    }
+
+    // The following functions are overrides required by Solidity.
+
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        override(ERC1155, AccessControl)
+        returns (bool)
+    {
+        return super.supportsInterface(interfaceId);
```

Comment:
super calling wrong parent. And here it should support both ERC1155 and AccessControl interfaces instead of just one

Reply:
This is a simple ERC1155 contract generated from the [open zeppelin wizard](https://docs.openzeppelin.com/contracts/4.x/wizard) 
supporting
* mintable
* updateable uri's
* roles

See this screenshot
![erc1155wizard](https://user-images.githubusercontent.com/1572027/191173453-1762fca1-30f7-4ce4-9c21-fb0c5cad8130.png)

If we need to support above requirements `ERC1155 and AccessControl interfaces instead of just one` we can manually write the mock contracts.

Reply:
they need to fix it

Reply:
As stated above I have corrected the code for our mock Tokens.
I have also raised https://github.com/OpenZeppelin/contracts-wizard/issues/183 for Open Zeppelin to fix the wizard.